recipes/pyside6qtads/meta.yaml

```diff
@@ -0,0 +1,64 @@
+ {% set version = "4.0.1" %}
+package:
+  name: pyside6qtads
```

Comment:
I  would like to hear @jschueller's opinion on the naming here.

Reply:
ok for me as the python import name is "PySide6QtAds",
however for the name of the c++ package "qt6-ads" I'd prefer something closer to the upstream name (Qt Advanced Docking System) because one could confuse it with a subpackage from the Qt project

Reply:
There is already a package for PyQt bindings for this library in conda-forge : https://github.com/conda-forge/pyqtads-feedstock
For consistency, I could rename Pyside6 bindings `pyside6ads' and Pyside2 bindings `pyside2ads'. What is the best for you?